urls/_2021.py

```diff
@@ -19,6 +19,16 @@
     'distro/core/qiime20212-BUILDID.zip':
         'https://qiime2.org',
 
+    # 2021.4 DISTRO
+    'distro/core/qiime2-2021.4-py36-osx-conda.yml':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2021.4/release/qiime2-2021.4-py36-osx-conda.yml',
+    'distro/core/qiime2-2021.4-py36-linux-conda.yml':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2021.4/release/qiime2-2021.4-py36-linux-conda.yml',
+    'distro/core/2021.4':
+        'https://qiime2-data.s3-us-west-2.amazonaws.com/distro/core/qiime2021-1614815453.zip',
+    'distro/core/qiime20214-1614815453.zip':
+        'https://qiime2-data.s3-us-west-2.amazonaws.com/distro/core/qiime2021-1614815453.zip',
```

Comment:
Two separate issues here. First, this is in the wrong section - you put this in the 2021.4 distro section, but these files should be in 2021.2 (also, please move this "DISTRO" section _below_ the 2021.2 distro section, we order these chronologically.

Two, there is a typo in this filename up on AWS.

![Screen Shot 2021-03-04 at 10 32 35 AM](https://user-images.githubusercontent.com/274668/110004708-f75e5e80-7cd4-11eb-9f3a-27d249770540.png)

The filename should start with `qiime20212-...`

Reply:
Oops!  Sorry about that.  I fixed the typos.

> also, please move this "DISTRO" section below the 2021.2 distro section


`# 2021.4 DISTRO` is below `# 2021.2 DISTRO`.  Can you clarify what you mean about the ordering?  It looks good to me.  What am I missing?

Reply:
I misread the diff - it looked like you put the 2021.4 distro section above the 2021.2 distro section, but I just misread the diff.

Reply:
The diff did look funky.  Thanks!